src/components/Documents/DropzoneUploader.tsx

```diff
@@ -0,0 +1,104 @@
+import 'react-dropzone-uploader/dist/styles.css';
+import '../../static/styles/UploadDocs.scss';
+
+import { tsConstructorType } from '@babel/types';
+import { Steps } from 'antd';
+import React, { useCallback, useState } from 'react';
+import { withAlert } from 'react-alert';
+import { Card, Col, Container, Dropdown, DropdownButton, Row, Spinner } from 'react-bootstrap';
+import Dropzone from 'react-dropzone-uploader';
+import { Link } from 'react-router-dom';
+import uuid from 'react-uuid';
+
+import getServerURL from '../../serverOverride';
+import PDFType from '../../static/PDFType';
+import Role from '../../static/Role';
+import DocumentViewer from './DocumentViewer';
+
+interface Props {
+    alert: any,
+    userRole: Role,
+    pdfFiles: File[] | undefined,
+    updateFileList,
+    updateStep
+  }
+
+  interface State {
+    pdfFiles: File[] | undefined,
+    currentStep: number,
+    maxNumFiles: number
+  }
+
+class DropzoneUploader extends React.Component<Props, State> {
+  constructor(props) {
+    super(props);
+    this.state = {
+      pdfFiles: undefined,
+      currentStep: 0,
+      maxNumFiles: 5,
+    };
+    this.handleFileStateChange = this.handleFileStateChange.bind(this);
+    this.handleCurrentStepChange = this.handleCurrentStepChange.bind(this);
```

Comment:
@ilei2 do you know how to use react hooks? It might be worth to add this directly so that we don't have to deal with the headache of migration @nrod80

Reply:
Ok, I updated DropzoneUploader into a functional component